utils/content.ts

```diff
@@ -13,6 +13,19 @@ const workDirectory = path.join(process.cwd(), 'content', 'work');
 const notesDirectory = path.join(process.cwd(), 'content', 'notes');
 const articlesDirectory = path.join(process.cwd(), 'content', 'articles');
 
+export interface IContentData {
+  id: string;
+  contentHtml: string;
+  date: Date;
+  title: string;
+  previewImage?: string;
+  description?: string;
+  tags?: string[];
+  category?: string;
+  problem?: string;
+  techStack?: string[];
+}
```

Comment:
moved to `content.ts` file because of ESLint errors. See the [comment](https://github.com/upleveled/next-starter-peacock/pull/153/files#r1512832403)